src/Core/Api/Services/FetchDocumentService.php

```diff
@@ -58,7 +58,7 @@ public function fetch(string $url): ResponseDTO
         $this->loggerContract
           ->error(
             TranslationHelper::getLoggerKey('cannotCallWayfairAPI'), [
-              'additionalInfo' => ['url' => $url, 'accessToken' => $this->authService->getOAuthToken()],
+              'additionalInfo' => ['url' => $url],
```

Comment:
(Similar to my other comment) - is there a way to let us know that the token was set, without leaking its value?